test/steps/domain/1_given.js

```diff
@@ -19,6 +19,10 @@ import {
 	getFirstNumber,
 } from '../utils';
 
+export function aPublicKeyThatShouldBeVotedFor() {
```

Comment:
`getFirstQuotedString` or else simply alias this to the existing function (maybe makes more sense since we're reusing the plural `this.test.ctx.votePublicKeys`).

Reply:
Yes, thats what I actually wanted to do. Fixed already